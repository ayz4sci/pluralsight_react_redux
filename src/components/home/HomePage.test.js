import React from 'react';
import renderer from 'react-test-renderer';
import HomePage from './HomePage';

describe ('HomePage Test', () => {
    it('renders correctly using snapshot', () => {
        const tree = renderer.create(<HomePage />).toJSON();
        expect(tree).toMatchSnapshot();
    })
});