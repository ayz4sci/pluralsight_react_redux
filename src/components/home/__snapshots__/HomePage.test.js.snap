// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage Test renders correctly using snapshot 1`] = `
<div
  className="jumbotron"
>
  <h1>
    Pluralsight Administration
  </h1>
  <p>
    React, Redux and React Router in ES6 for ultra-responsive web apps.
  </p>
  <a
    className="btn btn-primary btn-lg"
    onClick={[Function]}
    style={Object {}}
  >
    Learn more
  </a>
</div>
`;
